Makefile

```diff
@@ -1,16 +1,22 @@
-.PHONY: all image package dist clean
+.PHONY: all image package dist deploy clean
 
-all: package
+all: image dist
 
 image:
 	docker build --tag amazonlinux:nodejs .
 
-package: image
-	docker run --rm --volume ${PWD}/lambda:/build amazonlinux:nodejs npm install --production
+awsconfigure:
+	docker run -it --rm --volume ${PWD}/../.aws:/root/.aws amazonlinux:nodejs aws configure
```

Comment:
I just checked that everything is working by setting AWS_ACCESS_KEY_ID, AWS_SECRET_ACCESS_KEY, AWS_DEFAULT_REGION
Then I realized having these environment variables we would need to adjust the `Makefile` that the *deploy* target would pass them to the container.
How would you do that? Would you agree to write:
1. `docker run --rm -e AWS_ACCESS_KEY_ID=${AWS_ACCESS_KEY_ID} -e AWS_SECRET_ACCESS_KEY=${AWS_SECRET_ACCESS_KEY} -e AWS_DEFAULT_REGION=${AWS_DEFAULT_REGION} --volume ${PWD}:/build amazonlinux:nodejs bin/deploy`
1. `docker run --rm --env-file ~/.aws_env_file --volume ${PWD}:/build amazonlinux:nodejs bin/deploy`

I would prefer the first however it also doesn't look very nice.